manifests/kustomize/cluster-scoped-resources/kustomization.yaml

```diff
@@ -6,3 +6,23 @@ bases:
 - ../base/argo/cluster-scoped
 - ../base/pipeline/cluster-scoped
 - ../base/cache-deployer/cluster-scoped
+
+resources:
+  - namespace.yaml
+
+# Used by Kustomize
+configMapGenerator:
+  - name: pipeline-cluster-scoped-install-config
+    env: params.env
+
+vars:
```

Comment:
I think you should also add a `namespace: xxx` here, otherwise, the generated configmap will be in default namespace.